src/builtin-addons/core/script-definition-provider.ts

```diff
@@ -72,9 +72,18 @@ class PathResolvers {
     const pathParts = pathName.split('/');
 
     pathParts.shift();
-    const appParams = [root, 'app', ...pathParts];
+    const mayBePathParams = [
+      [root, 'app', ...pathParts],
+      [root, 'tests', ...pathParts],
```

Comment:
no, we don't need tests here, if we need to resolve tests scope, likely it should be new registry-based feature

Reply:
as per discussion offline, I will create another function that resolves the imports directly